### ISSUE TITLE
3.0.8:  build fails

### ISSUE THREAD

**Issue by kloczek** [libical/libical#464]
gcc 11.0.0-0.11 from fedora rawhide.
BTW .. do you have any plans to make new release soon? Looks like alreaduy above is commited a lot of patches.
```
/home/tkloczko/rpmbuild/BUILD/libical-3.0.8/src/libical/icalrecur.c: In function 'icalrecurrencetype_rscale_supported_calendars':
/home/tkloczko/rpmbuild/BUILD/libical-3.0.8/src/libical/icalrecur.c:1021:59: error: 'FALSE' undeclared (first use in this function)
 1021 |     en = ucal_getKeywordValuesForLocale("calendar", NULL, FALSE, &status);
      |                                                           ^~~~~
/home/tkloczko/rpmbuild/BUILD/libical-3.0.8/src/libical/icalrecur.c:1021:59: note: each undeclared identifier is reported only once for each function it appears in
/home/tkloczko/rpmbuild/BUILD/libical-3.0.8/src/libical/icalrecur.c: In function 'initialize_rscale':
/home/tkloczko/rpmbuild/BUILD/libical-3.0.8/src/libical/icalrecur.c:1414:63: error: 'FALSE' undeclared (first use in this function)
 1414 |         en = ucal_getKeywordValuesForLocale("calendar", NULL, FALSE, &status);
      |                                                               ^~~~~
gmake[2]: *** [src/libical/CMakeFiles/ical.dir/build.make:311: src/libical/CMakeFiles/ical.dir/icalrecur.c.o] Error 1
```

**Comment by eudaldgr** [comment]
I have the same error under KISS Linux, that's seems to be solved in this commit [`a3308a2`](https://github.com/libical/libical/commit/a3308a23912bba2db654a8c456165c31888cc897).
Meanwhile you can apply this patch:

```patch
--- a/src/libical/icalrecur.c
+++ b/src/libical/icalrecur.c
@@ -143,6 +143,7 @@
 #include <stdlib.h>

 #if defined(HAVE_LIBICU)
+#include <stdbool.h>
 #include <unicode/ucal.h>
 #include <unicode/ustring.h>
 #define RSCALE_IS_SUPPORTED 1
@@ -1018,7 +1019,7 @@

     calendars = icalarray_new(sizeof(const char **), 20);

-    en = ucal_getKeywordValuesForLocale("calendar", NULL, FALSE, &status);
+    en = ucal_getKeywordValuesForLocale("calendar", NULL, false, &status);
     while ((cal = uenum_next(en, NULL, &status))) {
         cal = icalmemory_tmp_copy(cal);
         icalarray_append(calendars, &cal);
@@ -1411,7 +1412,7 @@
         }

         /* Check if specified calendar is supported */
-        en = ucal_getKeywordValuesForLocale("calendar", NULL, FALSE, &status);
+        en = ucal_getKeywordValuesForLocale("calendar", NULL, false, &status);
         while ((cal = uenum_next(en, NULL, &status))) {
             if (!strcmp(cal, rule.rscale)) {
                 is_hebrew = !strcmp(rule.rscale, "hebrew");
```

**Comment by winterz** [comment]
yes, this has been fixed in the 3.0 branch for a few weeks.

I'll try to get a new release (v3.0.9) out the door "soon"

**Comment by kloczek** [comment]
Thx :)
